R/process_extract_home_care.R

```diff
@@ -66,7 +66,7 @@ process_extract_home_care <- function(
     # remove cost variables not from current year
     dplyr::select(-(tidyselect::contains("hc_cost_2"))) %>%
     # create cost total net
-    dplyr::mutate(cost_total_net = rowSums(dplyr::across(tidyselect::contains("hc_costs_q"))))
+    dplyr::mutate(cost_total_net = rowSums(dplyr::across(tidyselect::contains("hc_cost_q"))))
```

Comment:
We might as well swap this to use `pick` whilst making this change. Using `across` without a function will be deprecated at some point.
```suggestion
    dplyr::mutate(cost_total_net = rowSums(dplyr::pick(tidyselect::contains("hc_cost_q"))))
```